src/server/routes/projects.js

```diff
@@ -161,24 +161,23 @@ var applyAspectRatio = function (thumbnail, aspectRatio) {
 module.exports = [
     {
         Service: 'saveProject',
-        Parameters: 'socketId,overwrite,projectName',
+        Parameters: 'roleName,projectName,currentProjectName,ownerId,overwrite,srcXml,mediaXml',
```

Comment:
what do projectName and currentProjectName refer to? is this for the saveAs functionality?